## <center>Hypothesis Testing3</center>

#### Problem Statement:
<code style="background:lightblue;color:black">Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.</code>

In [1]:
#Importing the necessary libraries for the model
import pandas as pd
from scipy import stats
from scipy.stats import chi2_contingency

In [2]:
#Importing the csv file and having a glance at the data
buyer=pd.read_csv('BuyerRatio.csv')
buyer

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [3]:
#Listing the dataframe of the dataset
buyer.set_index(['Observed Values'],inplace=True)
buyer_data=buyer.values.tolist()
buyer_data

[[50, 142, 131, 70], [435, 1523, 1356, 750]]

In [4]:
#Finding the pvalue
stats,p,dof,expected=chi2_contingency(buyer_data)
pvalue=(stats,p,dof,expected)[1]
pvalue

0.6603094907091882

In [5]:
#Printing the output to know wether to accept the null hypothesis or not
alpha=0.05
print("p value is",pvalue)
if p<= alpha:
    print('Dependent(reject H0)')
else:
    print('Independent(H0 holds true)')
    

p value is 0.6603094907091882
Independent(H0 holds true)


#### Insights Drawn:
> Here H0: Proportions of male and female buyer ratio are all same<br>  H1:Proportions of male and female buyer ratio are all not same.<br> Hence From the pvalue above it is evident that we are accepting the null hypothesis since Pvalue=0.66>0.05,so all the proportions of the male and female ratio are same.